test/rql_test/test-runner

```diff
@@ -30,7 +30,7 @@ try:
     openFileLimit = int(os.getenv('OPEN_FILE_LIMIT', 512))
     if resource.getrlimit(resource.RLIMIT_NOFILE)[0] < openFileLimit:
         resource.setrlimit(resource.RLIMIT_NOFILE, (openFileLimit, openFileLimit))
-except Exception, e:
+except Exception as e:
```

Comment:
this syntax is accepted in Python 3 and 2.6+, whereas the comma is only accepted in 2.x